Instruments/gds.py

```diff
@@ -265,10 +265,9 @@ def upto_hashtag():
         logger.debug(strm("acquisition parameters", param))
         data_array *= float(param.pop("Vertical Scale"))
         data_array *= 5 * 1.032  # this is empirical
-        if not all(np.isfinite(x_axis)):
+        if not np.all(np.isfinite(x_axis)):
             raise ValueError(
-                "your x axis is not finite!! len(data_array) is %s and"
-                " 'Sampling Period' is %s"
+                "your x axis is not finite!! len(data_array) is %s and 'Sampling Period' is %s"
```

Comment:
this was probably done by black or something -- I would just convert to an f-string